proposals/parameterless-struct-constructors.md

```diff
@@ -116,10 +95,33 @@ A `base()` initializer is disallowed in struct constructors.
 
 The compiler will not emit a call to the base `System.ValueType` constructor from any struct instance constructors including explicit and synthesized parameterless constructors.
 
+### `record struct`
+If a `record struct` does not contain a primary constructor nor any instance constructors, and the `record struct` has field initializers, the compiler will synthesize a `public` parameterless instance constructor.
+```csharp
+record struct R0;                      // no parameterless .ctor
+record struct R1 { int F = 42; }       // synthesized .ctor: public R1() { F = 42; }
+record struct R2(int F) { int F = F; } // no parameterless .ctor
+```
+
+A `record struct` with an empty parameter list will have a parameterless primary constructor.
+```csharp
+record struct R3();                // primary .ctor: public R3() { }
+record struct R4() { int F = 42; } // primary .ctor: public R4() { F = 42; }
+```
+
+An explicit parameterless constructor in a `record struct` must call the primary constructor.
+```csharp
+record struct R5(int F)
+{
+    public R5() { } // error: must call 'this(string S)'
+    public int F =  F;
+}
+```
+
 ### Fields
-The synthesized parameterless constructor will zero fields rather than calling any parameterless constructors for the field types.
+The synthesized parameterless constructor will zero fields rather than calling any parameterless constructors for the field types. No warnings are reported that field constructors are ignored.
```

Comment:
> The synthesized parameterless constructor

Does this refer to the same thing as the "default value type constructor"?

Reply:
Yes. I've changed this to "implicitly-defined parameterless constructor" to match the "implicitly has a parameterless instance constructor" earlier (which I believe is the only other reference.